readme.md

```diff
@@ -1,6 +1,7 @@
 # customize-cra
 
-This project piggybacks on [`react-app-rewired`](https://github.com/timarney/react-app-rewired/) to customize create-react-app for version 2.0 and higher.
+This project provides a set of utilities to customize the Create React App v2
+configurations leveraging [`react-app-rewired`](https://github.com/timarney/react-app-rewired/) core functionalities.
```

Comment:
Edited to make it clearer the purpose of the project, before it looked like it was a replacement of react-app-rewired until you read the "how to install" section.